packages/big-design/src/components/Tooltip/spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+import { Tooltip } from './Tooltip';
+
+test('passes id to tooltip wrapper', async () => {
+  const { getByText, findByText } = render(
```

Comment:
I think we should start using `screen` instead of reading stuff from here. afaik that is the recommended way nowadays

Reply:
I thought about that, but for now I'm just using the existing pattern. There's some other things, like `user-event` we'd want to use too. But we should group that in a different PR.

Reply:
I'd disagree with that, since doing it incrementally where we can will lower the burden of changing stuff further down the road. i.e. this spec file is new but it already needs a refactor. Not blocking tho, up to you

Reply:
Good thought, updated 👍